packages/migrate/src/utils/unixSocket.ts

```diff
@@ -0,0 +1,17 @@
+import { DatabaseCredentials } from '@prisma/internals'
+
+const simpleUnixPathPattern = /^\.{0,2}\//
+
+export function getSocketFromDatabaseCredentials(credentials: DatabaseCredentials): string | null {
+  if (credentials.type === 'postgresql') {
```

Comment:
This also affects `cockroachdb`

> ```
> postgres://root@?host=/path/to/client&port=26257
> ```
> [source](https://www.cockroachlabs.com/docs/stable/connection-parameters#example-uri-for-a-unix-domain-socket)

Reply:
Okay, I've made that update. I also noticed there is an alternate value for PostgreSQL in the list of connection types, `postgres`, and I handled that one as well.